Source/environment.py

```diff
@@ -0,0 +1,137 @@
+    text = setUpE(screen, font, descriptions)
+    return [screen, text]
+
+def setUpE(screen, font, descriptions):
+    '''This code is to load the basic backscreen for the program.
+    setUpE is NOT to be run whenever the user clicks something, just at beginning.'''
+    #loading/scaling images (loads as S___ for start, scaled as F___ for final)
+    Sbee_Icon = pygame.image.load('Source/Course_Icons/bees.png')
+    Fbee_Icon = pygame.transform.scale(Sbee_Icon, (108, 100))
+    SENM_Icon = pygame.image.load('Source/Course_Icons/elec.png')
+    FENM_Icon = pygame.transform.scale(SENM_Icon, (108, 100))
+    SFOCS_Icon = pygame.image.load('Source/Course_Icons/focs.png')
+    FFOCS_Icon = pygame.transform.scale(SFOCS_Icon, (108, 100))
+    SFunRobo_Icon = pygame.image.load('Source/Course_Icons/robo.png')
+    FFunRobo_Icon = pygame.transform.scale(SFunRobo_Icon, (108, 100))
+    SMatSci_Icon = pygame.image.load('Source/Course_Icons/mats.png')
+    FMatSci_Icon = pygame.transform.scale(SMatSci_Icon, (108, 100))
+    SMechProto_Icon = pygame.image.load('Source/Course_Icons/mech.png')
+    FMechProto_Icon = pygame.transform.scale(SMechProto_Icon, (108, 100))
+    #basic background
+    pygame.draw.rect(screen, (255, 255, 255), (0,10,1000,440))
+    #icon buttons
+    pygame.draw.rect(screen, (255, 255, 255), (51,550,108,100))
+    pygame.draw.rect(screen, (255, 255, 255), (209,550,108,100))
+    pygame.draw.rect(screen, (255, 255, 255), (367,550,108,100))
+    pygame.draw.rect(screen, (255, 255, 255), (525,550,108,100))
+    pygame.draw.rect(screen, (255, 255, 255), (683,550,108,100))
+    pygame.draw.rect(screen, (255, 255, 255), (841,550,108,100))
+    #icon display
+    screen.blit(Fbee_Icon,(51,550))
+    screen.blit(FENM_Icon,(209,550))
+    screen.blit(FFOCS_Icon,(367,550))
+    screen.blit(FFunRobo_Icon,(525,550))
+    screen.blit(FMatSci_Icon,(683,550))
+    screen.blit(FMechProto_Icon,(841,550))
+    #render text
+    text = []
+    text.append(font.render("Sustainable Beekeeping: %s" % descriptions[1], 1, (255,255,255)))
+    text.append(font.render("Electricity and Magnetism: %s" % descriptions[3], 1, (255,255,255)))
+    text.append(font.render("Fundamentals of Computer Science: %s" % descriptions[2], 1, (255,255,255)))
+    text.append(font.render("Fundamentals of Robotics: %s" % descriptions[5], 1, (255,255,255)))
+    text.append(font.render("Material Science: %s" % descriptions[4], 1, (255,255,255)))
+    text.append(font.render("Mechanical Prototyping: %s" % descriptions[0], 1, (255,255,255)))
+    #Reload display / return font
+    pygame.display.flip()
+    return text
+
+def mouseClicks(order, BUTTON, screen):
+    '''This function looks for the click of a mouse and queues an event based on where the mouse is'''
+    (x,y) = pygame.mouse.get_pos()
+    clicked_path = 'Source/Course_Icons/%s_clicked.png'
+    if y >= 550 and y<= 655 and any(pygame.mouse.get_pressed()):
+        if x >= 51 and x<= 159 and 'bees' not in order:
```

Comment:
Where do these numbers come from? Can they come from a list of entities and their locations?